doc/classes/OS.xml

```diff
@@ -527,7 +534,8 @@
 			<argument index="0" name="pid" type="int">
 			</argument>
 			<description>
-				Kill a process ID (this method can be used to kill processes that were not spawned by the game).
+				Kill (terminate) a process ID, such as one returned by [method execute] in non-blocking mode.
```

Comment:
This is bad worded IMO. It kills the process identified by a process ID, it does not kill a process ID. The process ID did nothing wrong.

Reply:
Right, fixed :)